charts/operator-wandb/templates/_kafka.tpl

```diff
@@ -51,4 +50,16 @@ Return kafka broker url port
 */}}
 {{- define "wandb.kafka.brokerPort" -}}
 {{- print .Values.global.kafka.brokerPort -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+
+{{/*
+Return the kafka topic name for run-updates-shadow
+*/}}
+{{- define "wandb.kafka.runUpdatesShadowTopic" -}}
+{{- if eq .Values.global.kafka.runUpdatesShadowTopic "" -}}
+{{ printf "%s-%s" .Release.Name "run-updates-shadow" | trunc 63 | trimSuffix "-" }}
+{{- else -}}
+{{ .Values.global.kafka.runUpdatesShadowTopic }}
+{{- end -}}
+{{- end -}}
```

Comment:
To keep things manageable, let's use predefined values instead customer specific ones. Customers likely won't interact with this directly, and it'll make our Helm chart easier to maintain. We can always add it later, but we cannot remove options later

```suggestion
{{/*
Return the kafka topic name for run-updates-shadow
*/}}
{{- define "wandb.kafka.runUpdatesShadowTopic" -}}
{{ printf "%s-%s" .Release.Name "run-updates-shadow" | trunc 63 | trimSuffix "-" }}
{{- end -}}
```